contributing/topics/guide-new-fields-to-resource.md

```diff
@@ -109,89 +109,84 @@ A feature flag is essentially a function that returns a boolean and allows the p
 As an example, let's deprecate and replace the property `enable_compression` with `compression_enabled`.
 
 ```go
-Schema: map[string]*pluginsdk.Schema{
-    ...
-    "enable_compression": {
-        Type:     pluginsdk.TypeBool,
-        Optional: true,
-    },
+func (r ExampleResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"enable_compression": {
+			Type:	 pluginsdk.TypeBool,
+			Optional: true,
+		},
 }
 ```
 
 After deprecation the schema might look like the example below.
 
 ```go
-func resource() *pluginsdk.Resource {
-    resource := &pluginsdk.Resource{
-        ...
-        Schema: map[string]*pluginsdk.Schema{
-        	...
-
```

Comment:
This example is for an untyped resource which is still relevant given we have hundreds of these in the provider. It's great to add an equivalent example for a typed resource, but I don't think we should be removing the untyped examples yet. Can you please restructure this section into something like

- Here is what a deprecation would look like for an untyped resource
- Here is what a deprecation would look like for a typed resource